### PR TITLE
refactor: use columnadjuster from common repo

### PR DESCRIPTION
**Pull request by haxxmaxx**
Consuming the component moved here https://github.com/qlik-oss/nebula-table-utils/pull/75

### CODE REVIEW

**Comment by haxxmaxx** [comment]
I have no idea why codeclimate is picking up the issues in use-column-width at this point, I haven't introduced those issues

**Review by cbt1 — APPROVED** [review]
LGTM

File: src/pivot-table/components/cells/DimensionTitleCell.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { useStyleContext } from "../../contexts/StyleProvider";
 import type { GetHeaderCellsIconsVisibilityStatus, OverrideLeftGridWidth } from "../../hooks/use-column-width";
 import { useHeadCellDim } from "../../hooks/use-head-cell-dim";
 import { baseCellStyle, getBorderStyle } from "../shared-styles";
-import ColumnAdjuster from "./ColumnAdjuster";
+import ColumnAdjuster from "./ColumnAdjusterWrapper";
```
Comment (a-m-dev) [inline src/pivot-table/components/cells/DimensionTitleCell.tsx:14]: I think it's better to keep using `ColumnAdjusterWrapper` instead, 
if we try to search ColumnAdjuster -> it refers to 2 things now: 
1. this one (which wraps the actual adjuster from shared lib)
2. the one from shared lib

Same applies for `DimentionsValue/index` and the test file as well!

File: src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          {children}
+        </TestWithProvider>
+      ),
+    });
+
+  afterEach(() => jest.clearAllMocks());
+
+  test("should render ColumnAdjuster", () => {
+    renderAdjuster();
+    // TODO: import class string
```
Comment (a-m-dev) [inline src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx:32]: oops, 
maybe you are missing this: https://github.com/qlik-oss/nebula-table-utils/blob/0359f594fd3f695e373e5b05046b7822c792516f/src/constants/index.ts#L26

**Review by a-m-dev — APPROVED** [review]
LGTM 🚀

File: src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx (hunk 2 above)
Comment (haxxmaxx) [reply src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx:32]: good catch! it should be imported from the shared lib

File: src/pivot-table/components/cells/DimensionTitleCell.tsx (hunk 1 above)
Comment (haxxmaxx) [reply src/pivot-table/components/cells/DimensionTitleCell.tsx:14]: you mean the name of the import?

File: src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx (hunk 2 above)
Comment (haxxmaxx) [reply src/pivot-table/components/cells/__tests__/ColumnAdjusterWrapper.test.tsx]: done

**Pull request merged**